### DISCUSSION TITLE
Creating resource for Spatie Laravel Tags

### DISCUSSION THREAD

**Discussion by bartclaeys** [filamentphp/filament#5323] [General]
I installed Spatie Laravel Tags and Filament Tags plugin packages, but unfortunately this didn't come with a resource to manage (view, edit, delete) the tags and tag types. I don't know how to start to create a resource for this.

Should I start with creating a new Tag model or should I simply point to the Tag model Spatie already has?

**Comment by danharrin** [comment]
Point to Spatie's model, and use the translatable plugin too for the resource since the tags package uses that

**Reply by bartclaeys** [reply]
Cool, it worked! What would the translatable plugin do? Right now I am seeing this:

<img width="975" alt="Screen Shot 2022-12-23 at 09 36 07" src="https://user-images.githubusercontent.com/167579/209378295-a71226fb-0793-4ad9-bb0c-4983a32c13c3.png">

**Reply by ssglopes** [reply]
Hi how did you manage to get that screen? I generate a tag page myself using `artisan make:filement-resource Tag` but it seems this should come straight already from the package itself.. Thanks
